service/src/main/resources/application.yml

```diff
@@ -54,6 +54,8 @@ spring:
       static-locations: classpath:/api/
 
 management:
+  server:
+    port: 9098
```

Comment:
Don't want to leave on default 8080 port because that will allow anyone to access the metrics. This port will not be proxied outside the pod (as far as I know!).